### DISCUSSION TITLE
Daily download quarantined by Antivirus

### DISCUSSION THREAD

**Discussion by MrTAB** [Brewtarget/brewtarget#650] [Q&A]
Hello,

I was upgrading from 2.2 to 2.3, but 2.3 introduces a bug in hop timing so I went for the daily build as I saw that this issue was fixed.
I got the daily build from Actions, and have tried the 23rd and 24th of July 2022 builds, both to the following effect:
My anti-virus quarantines .dlls claiming they are a Win32:MalOB-IJ threat.

This could be a false positive from Avast, in which case I'll tell them so, but as its the daily build I thought I should reach out in case the build pipeline has been compromised. (There is always the possibility of it being something on my end, but i did install 2.3 immediately before, from http://www.brewtarget.org/download.html)

Would someone who has say original dependency .dlls (e.g. the original QtMultimedia 5.15.3.0) be able to compare the true source against those produced by the installer?

Thanks,
~Tony

**Comment by matty0ung** [comment]
Sorry to hear you're having problems.  I don't have a Windows build set up, so I can't help directly but @mattiasmaahl may be able to shed some insight as and when he has some time.

AFAICT no recent changes have been made to the Windows build -- see control file https://github.com/Brewtarget/brewtarget/blob/develop/.github/workflows/windows.yml which was last modified 12 months ago.  But we do depend on a lot of third-party libraries, so it's at least conceivable that there could be a problem with one of them, though most of them are so mainstream and widely-used that it would be something of a surprise.

If you by any chance have a little time yourself, you might be able to narrow things down by following the instructions at https://github.com/Brewtarget/brewtarget/wiki/Building-on-Windows. You don't necessarily need to get as far as building Brewtarget itself, but I was thinking that, if, as you install the build dependencies, Avast suddenly pops up and says I don't like library X, then that would be a massive step forward in pinpointing what the problem is.

**Comment by MrTAB** [comment]
Thanks for getting back to me Matty,

I was thinking, it is extremely unlikely that I'd be the first person to find e.g. something in Qt.

I'll not be able to try a build setup today but if I do later I will come back with the results.
